src/sentry/integrations/vercel/client.py

```diff
@@ -46,3 +50,19 @@ def create_deploy_webhook(self):
         }
         response = self.post(self.WEBHOOK_URL, data=data)
         return response
+
+    def get_env_vars(self, vercel_project_id):
+        return self.get(self.GET_ENV_VAR_URL % vercel_project_id)
+
+    def get_secret(self, name):
+        return self.get(self.GET_SECRET_URL % name.lower())["uid"]
```

Comment:
What catches the errors when these requests fail or 'succeed' with an HTML page?

Reply:
@markstory Those cases will blow up the function and generate a 500 error for the client and send it to Sentry. I think that's good enough here.